tools/generators/epic-generator/index.ts

```diff
@@ -0,0 +1,186 @@
+import { getProjects, stripIndents, Tree } from '@nrwl/devkit';
+import { access, readFileSync } from 'fs';
+import { execSync } from 'child_process';
+import fetch from 'node-fetch';
+import { EpicGeneratorSchema } from './schema';
+
+function validateSchema(schema: EpicGeneratorSchema) {
+  if (!schema.repository.match(/[A-z-]+\/[A-z-]+/)) {
+    throw new Error('Must provide a valid repository for the issue');
+  }
+
+  if ((schema.title || '').trim() === '') {
+    throw new Error('Must provide a valid title for the issue');
+  }
+
+  return {
+    repository: schema.repository,
+    title: schema.title,
+    message: schema.message || '*Description to be added*',
+  };
+}
+
+const checkAuthentication = async () => {
+  const authStatus = execSync(`gh auth status`).toString();
+
+  if (authStatus.search('You are not logged into any GitHub hosts') > 0) {
+    throw new Error('You are not logged into GitHub CLI (gh).');
+  }
+};
+
+interface Package {
+  name: string;
+  folder: string;
+  owners?: string[];
+}
+
+interface Ownership {
+  path: string;
+  owners: string[];
+}
+
+interface MigrationIssue {
+  issueUrl?: string;
+  assignee?: string;
+  packages: Package[];
+}
+
+interface MigrationIssueMap {
+  [key: string]: MigrationIssue;
+}
+
+const getConvergencePackages = (tree: Tree) => {
+  const projects = getProjects(tree);
+
+  let convergencePackages: Package[] = [];
+  projects.forEach((project, key) => {
+    if (project.projectType === 'library' && project.tags?.some(tag => tag === 'vNext')) {
```

Comment:
I think we have an `isConvergedPackage` utility that's already used a few times, might be worth updating that and using it if tags is the way to go

Reply:
Should generators depend on `scripts` stuff though? Probably the way to go would be to have some common utilities in generators once we need to reuse. Wdyt?

Reply:
> Should generators depend on scripts stuff though?

nope, pls dont do that ever :) we have isV8 and isConverged in tools/utils - that's the source of truth